src/extension.ts

```diff
@@ -1097,6 +1098,24 @@ async function getLogsForPod(pod: PodSummary, follow?: boolean) {
     }
 }
 
+async function getEvents(follow: kuberesources.EventDisplayMode, explorerNode?: explorer.ResourceNode) {
+    let currentNS;
+
+    if (explorerNode) {
+        currentNS = explorerNode.id;
+    } else {
+        // defaulting to current namespace in kube config
+        currentNS = await kubectlUtils.currentNamespace(kubectl);
+    }
+
+    let cmd = ` get events --namespace ${currentNS}`;
+    if (kuberesources.EventDisplayMode.Follow === follow) {
+        cmd += ' -w';
+    }
+
+    kubectl.invokeInSharedTerminal(cmd);
```

Comment:
We probably don't want to use the shared terminal in `follow` mode, because it blocks the shared terminal from being used for other commands such as logs.  Follow mode should start a new terminal with a suitable descriptive name; see the `kubectl.invokeInNewTerminal` method.

This is also a bug in the existing `Follow Logs` command (and I appreciate you following the pattern; sorry we missed this on the logs code).  If you can address it here then we can backport your fixes for both the terminal blocking issue and the Boolean parameter issue onto the existing logs code.

Reply:
I'll file an issue for this